app/views/shared/_stream_element.mobile.haml

```diff
@@ -29,3 +29,6 @@
 
     - if defined?(expanded_info) && expanded_info
       = render partial: "shared/post_stats", locals: {post: @post}
+
+    .add-comment-switcher{class: ('hidden' unless defined?(expanded_info) && expanded_info)}
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.